client/db/bolt/db.go

```diff
@@ -229,7 +229,7 @@ func (db *BoltDB) CreateAccount(ai *dexdb.AccountInfo) error {
 		return fmt.Errorf("zero-length EncKey not allowed")
 	}
 	return db.acctsUpdate(func(accts *bbolt.Bucket) error {
-		acct, err := accts.CreateBucket([]byte(ai.Host))
+		acct, err := accts.CreateBucket([]byte(ai.Host + "-" + string(ai.DEXPubKey.Serialize()) + "-" + string(ai.ClientPubKey.Serialize())))
```

Comment:
As far as creating the bucket key, it's a bit awkward converting the keys to strings just to do string concatenation and then back to a byte slice.  Really the key doesn't need to be based on a string at all afaik, so the `"-"` delimiters aren't needed.

It's more cumbersome than a one-liner with string concatenation, but I'd suggest a small function to make the key, maybe:

```go
func accountInfoKey(ai *dexdb.AccountInfo) []byte {
	acctKey := make([]byte, len(ai.Host), len(ai.Host)+2*dexdb.PubKeySize)
	copy(acctKey, []byte(ai.Host))
	acctKey = append(acctKey, ai.DEXPubKey.Serialize()...)
	acctKey = append(acctKey, ai.ClientPubKey.Serialize()...)
	return acctKey
}
```

where dexdb defines `const PubKeySize = secp256k1.PubKeyBytesLenCompressed`